package.json

```diff
@@ -9,7 +9,7 @@
     "test": "jest",
     "start": "NODE_ENV=production node server.js",
     "format-staged": "pretty-quick --staged --no-restage --bail",
-    "format-check": "pretty-quick --check --verbose",
+    "format-check": "prettier --check '{.,pages/**,src/**,static/**}/*.{js,jsx,md}'",
```

Comment:
is it better to omit extensions? and let it figure it out automatically?

Reply:
No, it tries to check literally all files (including Git-ignored ones and `.gitignore` itself) and throws a bunch of `No parser could be inferred for file` errors. You can try it yourself:

```console
$ yarn prettier --check '{.,pages/**,src/**,static/**}/*'
```

Funily, it checks `.prettierrc` (with YAML parser I guess?) and finds errors in it.

Reply:
👍